docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,428 @@
+
+```sql
+SELECT
+    event_date,
+    count(*) as count,
+    user
+FROM
+    session_start
+WHERE
+    CARDINALITY(access_requests) IS NULL
+AND
+    proto='ssh'
+AND
+    with_mfa IS NULL
+AND
+    impersonator IS NULL
+AND
+    trusted_device_device_id IS NULL
+GROUP BY
+    event_date,
+    proto,
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Set up access requests, device trust and per-session MFA.
+
+### Kubernetes API calls with weak security
+
+The Following query identifies Kubernetes sessions with weak security,
+such as sessions with missing access requests, MFA, impersonation, and trusted device identification.
+
+```sql
+SELECT
+    event_date,
+    count(*) as count,
+    user
+FROM
+    kube_request
+WHERE
+    CARDINALITY(access_requests) IS NULL
+AND
+    with_mfa IS NULL
+AND
+    impersonator IS NULL
+AND
+    trusted_device_device_id IS NULL
+GROUP BY
+    event_date,
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Set up access requests, device trust and per-session MFA.
+
+### Privileged Postgres sessions
+
+The Following query identifies privileged PostgreSQL sessions initiated by the 'postgres' user.
+
+```sql
+SELECT
+	event_date,
+	COUNT(*) AS count,
+	user
+FROM
+    db_session_start
+WHERE
+    db_protocol='postgres' and db_user='postgres'
+GROUP BY
+    event_date,
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:** Downgrade database connections to less privileged database user.
+
+### Kube Execs
+
+The Following query identifies Kubernetes exec usage.
+
+```sql
+SELECT
+    event_date,
+    count(*) as count,
+    user
+FROM
+    exec
+WHERE
+    proto='kube'
+GROUP BY
+    event_date,
+    proto,
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Eliminate usage of kube exec.
+
+
+### Long lived certificates
+
+The Following query identifies long-lived certificates, where the certificate expiration time is greater than 1 day.
+
+```sql
+SELECT DISTINCT
+    event_date,
+    COUNT(*) AS count,
+    identity_user AS user
+FROM
+    cert_create
+WHERE
+    from_iso8601_timestamp(identity_expires) - from_iso8601_timestamp(time) > INTERVAL '1' DAY
+GROUP BY
+    event_date,
+    identity_user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Use short lived certificates less than a working day. Check out Machine ID to get short lived certificates for your automation.
+
+### Long-lived join tokens
+
+The Following query identifies long-lived join tokens where the token expiration time is greater than 1 day.
+
+```sql
+SELECT
+    event_date,
+    COUNT(*) as count,
+    node_name,
+    host_id
+FROM
+    instance_join
+WHERE
+    FROM_ISO8601_TIMESTAMP(token_expires) - FROM_ISO8601_TIMESTAMP(time) > INTERVAL '1' DAY
+GROUP BY
+    event_date,
+    node_name,
+    host_id
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Use short-lived tokens to reduce risk of compromise or AWS/GCP/Azure joining when possible.
```

Comment:
```suggestion
**Suggestion:**  Use short-lived tokens to reduce risk of compromise or delegated joining when possible.
```

Bonus points for linking to Hugo's new joining page.

Reply:
Done, Thanks for suggestion.